exo-go/bin/spec

```diff
@@ -3,5 +3,5 @@ set -e
 
 bin/lint
 bin/build
-bin/unit-test
 bin/feature-test
+bin/unit-test
```

Comment:
Still investigating. Apparently, there is a bug in the implementation of `exo-clean` feature tests.